### ISSUE TITLE
The text associated with this error code could not be found (0x80D05011)

### ISSUE THREAD

**Issue by D-Maxwell** [files-community/Files#13865]
### Description

Upon installing the app through the classic installer (as I have no access to the Microsoft Store), the installation fails with the following exit code, indicating that there is an unknown undocumented error :
```
Deployment failed with HRESULT: 0x80D05011, The text associated with this error code could not be
found.
Error message string not found
Appinstaller operation failed with error code 0x80D05011. Detail: Unknown error
NOTE: For additional information, look for [ActivityId] 51858f03-0274-0003-7c4f-de517402da01 in the Event Log or use
the command line Get-AppPackageLog -ActivityID 51858f03-0274-0003-7c4f-de517402da01
At line:1 char:1
+ Add-AppxPackage -AppInstallerFile https://cdn.files.community/files/s ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (https://cdn.fil...ge.appinstaller:String) [Add-AppxPackage], Exception
    + FullyQualifiedErrorId : DeploymentError,Microsoft.Windows.Appx.PackageManager.Commands.AddAppxPackageCommand
```

Note that this is obtained through the command ``Add-AppxPackage -AppInstallerFile https://cdn.files.community/files/stable/Files.Package.appinstaller`` provided in a previous issue.
The corresponding logs are as follows :
```
Time                      ID           Message
----                      --           -------
14/11/2023 11:40:55       603          Started deployment Add operation on a package with main parameter
                                       Files.Stable.exe and Options 0 and 0. See
                                       http://go.microsoft.com/fwlink/?LinkId=235160 for help diagnosing app
                                       deployment issues.
14/11/2023 11:40:55       465          error 0x8007000D: Opening the package from location Files.Stable.exe failed.
14/11/2023 11:40:55       497          Error 0x80070002: Opening the Msixvc package from location Files.Stable.exe
                                       failed. Please check whether the Msixvc support services are installed.
14/11/2023 11:40:55       403          error 0x8007000D: Failure to get staging session for:
                                       file:///C:/Users/maximdovergne/Downloads/Files.Stable.exe.
14/11/2023 11:40:55       404          AppX Deployment operation failed for package  with error 0x80073CF0. The
                                       specific error text for this failure is: error 0x8007000D: Opening the package
                                       from location Files.Stable.exe failed.
```

Running a local powershell install, ``Add-AppxPackage .\Files.Stable.exe``, results in the following :
```Deployment failed with HRESULT: 0x80073CF0, Package could not be opened.
error 0x8007000D: Opening the package from location Files.Stable.exe failed.
NOTE: For additional information, look for [ActivityId] 51858f03-0274-0000-9e06-c3517402da01 in the Event Log or use
the command line Get-AppPackageLog -ActivityID 51858f03-0274-0000-9e06-c3517402da01
At line:1 char:1
+ Add-AppxPackage .\Files.Stable.exe
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OpenError: (C:\Users\maximd...iles.Stable.exe:String) [Add-AppxPackage], FileNotFoundExc
   eption
    + FullyQualifiedErrorId : DeploymentError,Microsoft.Windows.Appx.PackageManager.Commands.AddAppxPackageCommand
```
```
Time                      ID           Message
----                      --           -------
14/11/2023 11:44:07       603          Started deployment AddFromAppInstallerOperation operation on a package with
                                       main parameter Files.Package.appinstaller and Options 0 and 0. See
                                       http://go.microsoft.com/fwlink/?LinkId=235160 for help diagnosing app
                                       deployment issues.
14/11/2023 11:44:08       651          Appinstaller operation failed with error code 0x80D05011. Detail: Unknown error
14/11/2023 11:44:08       404          AppX Deployment operation failed for package
                                       https://cdn.files.community/files/stable/Files.Package.appinstaller with error
                                       0x80D05011. The specific error text for this failure is: Appinstaller operation
                                       failed with error code 0x80D05011. Detail: Unknown error
```

### Steps To Reproduce

Download classic installer
Run
Crash

### Requirements

The app should install itself just like the Preview

### Files Version

...

### Windows Version

10.0.19044.2486

### Log File

none

**Comment by Josh65-2201** [comment]
Have you install a unofficial (like Atlas OS) or modified (like disabled windows update, removed the Microsoft store, used a debloat program/script etc.) version of Windows?

**Comment by D-Maxwell** [comment]
> Have you install a unofficial (like Atlas OS) or modified (like disabled windows update, removed the Microsoft store, used a debloat program/script etc.) version of Windows?

I have indeed manually got rid of the Microsoft Store app, as I only make use of WinGet, its derivatives, classic app installations, and AppX packages.

Windows Updates have also been disabled on this machine as they mess up registry tweaks I have made, and unpredictably reinstall bloatware ; although I wish to make it clear that I am not making use of any modified Windows 10 Roms.

------

What I find interesting to investigate however is the fact that I have ran similar tweaks on my desktop (as this is my laptop), yet I have had no issues whatsoever installing it there through the "Classic Installer".

Is there any service this installer relies on (and which the Preview does not, since that one seems to work just fine) that could've potentially been disabled as part of debloating this system of mine ?
